.github/workflows/test.yml

```diff
@@ -29,7 +29,8 @@ jobs:
       - name: Run ansible-lint
         uses: ansible/ansible-lint@main
         with:
-          requirements_file: requirements.yml
+          requirements_file: ../requirements.yml
```

Comment:
I was wondering if we should move `requirements.yml` into `src` instead so I looked up how a collection is structured. https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html tells me we should have a `galaxy.yml` in the same directory as `roles` and `playbooks`.

We can ignore it for now but where I'd like to head to is that we can point obsah at a collection and it can just do the right thing.